meson_options.txt

```diff
@@ -14,6 +14,7 @@ option('polkit', type: 'feature', description : 'PolKit support in daemon', depr
 option('gnutls', type: 'feature', description : 'GnuTLS support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('sqlite', type: 'feature', description : 'sqlite support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('lzma', type: 'feature', description : 'LZMA support', deprecated: {'true': 'enabled', 'false': 'disabled'})
+option('cbor', type: 'feature', description : 'CBOR support')
```

Comment:
Maybe expand this description some.  I don't think CBOR is a generally well known acronym like LZMA is.

Reply:
something like `CBOR (Concise Binary Object Representation) support` or something like `CBOR support (for coSWID and uSWID)`

Reply:
Yeah either of those